perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -193,14 +193,17 @@ sub bin_services_for_address {
     my $self = shift;
     my $property = shift;
 
-    my $site_services = $self->whitespace->GetSiteCollections($property->{uprn});
+    my @site_services;
```

Comment:
Looks like this works fine, but if I'm reading it right - you either have the child services, or if they're empty, the parent services, you never have both. So I think you didn't need to add the UPRN to every result, you only needed to note either the child or the parent UPRN depending on whether the child had services or not. Guess up to you if you want to simplify it or keep it as is.

Reply:
Good point! Have added a fixup commit to simplify.